PRFCS/prfc-1.md

```diff
@@ -13,6 +13,12 @@ A standard contract interface for fungible tokens allows more seamless interoper
 
 The below section lists the set of methods that all smart contracts that want to be PRFC 1-compliant must implement, as well as the behaviour that each method must exhibit. Required behaviour involves emitting certain events. These are also listed and described.
 
+## Notes
+
+- The following uses syntax from Rust (version 1.59.0).
+- The data type `PublicAddress` in this context is the type alias to 32-byte slice `[u8; 32]`. 
```

Comment:
- The data type `PublicAddress` in this context is the type alias to **a** 32-byte slice `[u8; 32]`.